tests/integration/plugins/test_maven.py

```diff
@@ -18,11 +18,47 @@
 import textwrap
 from pathlib import Path
 
+import pytest
 import yaml
-from craft_parts import LifecycleManager, Step
+from craft_parts import LifecycleManager, Step, errors
 
 
-def test_maven_plugin(new_dir, partitions):
+@pytest.fixture
+def use_mvnw(request):
+    if request.param:
+        yield request.param
+        return
+    source_location = Path(__file__).parent / "test_maven"
+    mvnw_file = source_location / "mvnw"
+    mvnw_file = mvnw_file.rename(f"{source_location}/mvnw.backup")
```

Comment:
why this rename? Shouldn't the wrapper be ignored if `maven-use-mvnw` is False?

Reply:
Yes you're right - however, in order to test cases where `maven-use-wrapper=True` without the `mvnw` file in the project, I've created this helper fixture. I am renaming it because I would like to reset it back once that particular test is done.
Would you know a cleaner way to approach this?
Thank you for the review!